README.md

```diff
@@ -263,6 +263,15 @@ $ sudo dnf copr enable carlwgeorge/ripgrep
 $ sudo dnf install ripgrep
 ```
 
+If you're an **openSUSE Leap 15.0** user, you can install ripgrep from the
```

Comment:
It looks like we already have openSUSE Tumbleweed instructions. Is there any way to collapse them into one section? What's the difference? (I don't know what the difference between them is.

Reply:
Leap and Tumbleweed are very different: the difference is verbosely explained in https://en.opensuse.org/Portal:Tumbleweed and https://www.opensuse.org/#Leap

Maybe we can create a super section called "openSUSE" and then introduce the two flavors to get ripgrep